client/src/utils/fetchData.js

```diff
@@ -1,15 +1,17 @@
 import axios from 'axios';
 
-const fetchData = async (path1, path2) => {
-  const [itemResponse, thumbnailsResponse] = await Promise.all([
+const fetchData = async (path1, path2, path3) => {
+  const [itemResponse, thumbnailsResponse, titleResponse] = await Promise.all([
     axios.get(path1),
     axios.get(path2),
+    axios.get(path3),
   ]);
   thumbnailsResponse.data.thumbnailImages.forEach((url, i) => {
     if (itemResponse.data.colors[i]) {
       itemResponse.data.colors[i].thumbnail = url;
     }
   });
+  itemResponse.data.title = titleResponse.data;
   return itemResponse.data;
```

Comment:
this is a great helper function!

Reply:
I actually decided I don't love factoring it out like I did into its own file..I kind of like what it does, but if one of those promises fails, the whole thing goes down...I actually think I prefer having it like yours is, but I'm not going to refactor now, cause it works.